mindsdb/integrations/handlers/ray_serve_handler/ray_serve_handler.py

```diff
@@ -37,9 +39,19 @@ def create(self, target: str, df: Optional[pd.DataFrame] = None, args: Optional[
         args['target'] = target
         self.model_storage.json_set('args', args)
         try:
-            resp = requests.post(args['train_url'],
-                                 json={'df': df.to_json(orient='records'), 'target': target},
-                                 headers={'content-type': 'application/json; format=pandas-records'})
+            print(pd.__version__)
+            if args.get('parquet', False):
+                buffer = io.BytesIO()
+                df.attrs['target'] = target
+                df.to_parquet(buffer)
```

Comment:
Please use  the read_parquet from the FileReader https://github.com/mindsdb/mindsdb/blob/bacf861958191fe53e62369f025ebe7a1e27a94c/mindsdb/integrations/utilities/files/file_reader.py#L303